ziti/cmd/edge/quickstart_automated_test.go

```diff
@@ -15,7 +15,6 @@ import (
 func TestEdgeQuickstartAutomated(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	_ = os.Setenv("ZITI_CTRL_EDGE_ADVERTISED_ADDRESS", "localhost") //force localhost
-	_ = os.Setenv("ZITI_ROUTER_NAME", "quickstart-router")
```

Comment:
I misunderstood this. It needs to be set here because this is the automated test which is not designed to accept vars from parent env, but rather must define all vars needed to test in a self-contained manner.